lib/fencing/st_output.c

```diff
@@ -109,7 +109,7 @@ stonith__full_history(pcmk__output_t *out, va_list args) {
 }
  
 PCMK__OUTPUT_ARGS("full-fencing-list", "crm_exit_t", "stonith_history_t *", "GListPtr", "gboolean", "gboolean")
-int
+static int
 stonith__full_history_xml(pcmk__output_t *out, va_list args) {
```

Comment:
Let's strip the prefix from these names as well. The goal is that the prefix gives a hint as to where and how the function is defined -- static functions defined in the same file shouldn't have a prefix.